File: Python/pattern5.py
# This is simple diamond pattern python-program (notice reduced 1 for loop in this code)
for i in range(0,11,1):

    for j in range(5,i,-1):
        print(" ",end="")

    for y in range(0,i,1):
        if i<=4:
            print("x",end=" ")
        else:
            break

    for x in range(5,i,1):
        print(" ",end="")
    
    for u in range(10,i,-1):
        if i>=5:
            print("x",end=" ")
            

    print("x",end="\n")

File: C/pattern5.c
// # This is simple diamond pattern c-program
#include <stdio.h>
int main(){
    
    for(int i = 1;i<=5;i++){

        for(int j = 5;j>=i;j--){
            printf(" ");
        }

        for(int k = 1;k<=i;k++){
                printf("x ");
            }

        printf("\n");
        }

    
	for (int t = 0;t<=5;t++){
		
        for(int l = 1;l<=t;l++){
            printf(" ");
        }

        for(int u = 5;u>=t;u--){
           
            printf("x ");
            
        }

		printf("\n");
	}
    return 0;
}